tests/test_library_refs.py

```diff
@@ -0,0 +1,13 @@
+import yaml
+
+
+def test_library_refs():
+    yaml_file = open("repositories.yaml", "r")
+    repositories = yaml.safe_load(yaml_file)
+    yaml_file.close()
+    libs = set(library["library_name"] for library in repositories["libraries"])
+    for app in repositories["applications"]:
+        if missing_libs := set(app["dependencies"]) - libs:
```

Comment:
looks like the installed version of flake8 doesn't like `:=`, so this needs to be split across two statements:
```suggestion
        missing_libs = set(app["dependencies"]) - libs
        if missing_libs:
```